web/client/utils/openlayers/__tests__/Icons-test.js

```diff
@@ -26,7 +26,6 @@ describe('Icons openlayers styles', () => {
         const shadow = styles[0];
         expect(shadow).toExist();
         const shadowImage = shadow.getImage();
-        expect(shadowImage.getAnchor()).toEqual([12, 12]);
```

Comment:
why removing this here and not also below
`expect(iconImage.getAnchor()).toEqual([18, 46]);`

Reply:
done